### PR TITLE
feat(github-actions): support jobs.<job_id>.container.image

### PR DESCRIPTION
**Pull request by douglasduteil**
## Changes:

Supported github actions syntax `jobs.<job_id>.container.image`

```
container:
  image: hadolint/hadolint:v1.19.0-alpine
```

see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainerimage

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/manager/github-actions/extract.ts [hunk 1]
```diff
@@ -13,6 +13,16 @@ export function extractPackageFile(content: string): PackageFile | null {
       continue; // eslint-disable-line no-continue
     }
 
+    const containerImageMatch = /^\s+image: ([^"]+)\s*$/.exec(line);
+    if (containerImageMatch) {
+      const [, currentFrom] = containerImageMatch;
+      const dep = getDep(currentFrom);
+      dep.depType = 'docker';
+      dep.versioning = dockerVersioning.id;
```
Comment (viceice) [inline lib/manager/github-actions/extract.ts:21]: @rarkins I think this is obsolete?

**Review by viceice — COMMENTED** [review]
I don't see any new test for it?

File: lib/manager/github-actions/extract.ts (hunk 1 above)
Comment (rarkins) [reply lib/manager/github-actions/extract.ts:21]: Yes, it is not necessary

**Comment by douglasduteil** [comment]
>  don't see any new test for it?

Test added @viceice :)

File: lib/manager/github-actions/extract.ts (hunk 1 above)
Comment (rarkins) [reply lib/manager/github-actions/extract.ts:21]: Please remove this line

**Review by rarkins — CHANGES_REQUESTED** [review]
Remove versioning.

**Comment by douglasduteil** [comment]
> Remove versioning.

Done @rarkins 👍

**Review by rarkins — APPROVED** [review]

**Comment by viceice** [comment]
@douglasduteil We had to revert this pr, see #8897

**Pull request merged**